internal/receiver/scriptedinputsreceiver/testdata/resource_logs/cpu.yaml

```diff
@@ -0,0 +1,4 @@
+resource_logs:
+  - scope_logs:
+      - logs:
+          - body: "<RE2(CPU    pctUser  pctSystem    pctIdle)>"
```

Comment:
Can this be expanded to demonstrate more content? attribute checks would be helpful too imo.

Reply:
The output looks like this:
CPU    pctUser  pctSystem    pctIdle
all       3.89        6.8       90.1

What do you suggest to test more?